integration_test/cleanup/test-cleanup.py

```diff
@@ -0,0 +1,77 @@
+import boto3
+import os
+from orca_shared.database import shared_db
+from sqlalchemy import text
+from aws_lambda_powertools import Logger
+from botocore.exceptions import ClientError
+
+
+LOGGER = Logger()
+
+# os.env keys
+recovery_bucket_name = os.environ["orca_RECOVERY_BUCKET_NAME"]
+source_bucket_name = os.environ["SOURCE_BUCKET_NAME"]
+db_connect_info_secret_arn = os.environ["DB_CONNECT_INFO_SECRET_ARN"]
+
+# grab the objects to be deleted
+s3 = boto3.client('s3')
+result = s3.list_objects_v2(
+    Bucket=source_bucket_name,
+    Prefix='test'
+)
+filenames = []
+for item in result['Contents']:
+    files = item['Key']
+    filenames.append(files)
+LOGGER.info(filenames)
+
+# Delete objects from source and recovery buckets
+for object in filenames:
+    response_source = s3.delete_object(
+        Bucket=source_bucket_name,
+        Key=object,
+    )
+    response_destination = s3.delete_object(
+        Bucket=recovery_bucket_name,
```

Comment:
This does assume that the files will have the same keys in each bucket. Fine for now, but it might be wise to future-proof by searching for all files present in the recovery bucket and appending them to a different list.

Reply:
so you are saying separately grab the files under "test" dir in a new list in recovery bucket and then do the deletion?

Reply:
updated with creating a new list for recovery bucket